### PR TITLE
fine-tuning software support

### PR DESCRIPTION
**Pull request by lmichael107**

### CODE REVIEW

**Comment by lmichael107** [comment]
@brianhlin or @djw8605 take a look and merge?

File: _about/computation.md [hunk 1]
```diff
@@ -8,20 +8,20 @@ Jobs run on the OSG will be able to execute on servers at numerous remote physic
...
-3. Software dependencies can be can be staged with the job, distributed via containers, or installed on the read-only distributed OASIS filesystem (which can also support software modules). Statically-linked binaries are ideal. However, dynamically linked binaries with standard library dependencies, built for 64-bit Red Hat Enterprise Linux (RHEL) version 6 or 7 will also work. OSG can support some licensed software (like Matlab, Matlab-Simulink, etc.) where compilation allows execution without a license, or where licenses still accommodate multiple jobs and are not node-locked.
+3. Software dependencies can be can be staged with the job, distributed via containers, or installed on the read-only distributed OASIS filesystem (which also supports OSG-curated software modules). Statically-linked binaries are ideal. However, dynamically linked binaries with standard library dependencies, built for 64-bit Red Hat Enterprise Linux (RHEL) version 6 or 7 will also work. OSG can support some licensed software (like Matlab, Matlab-Simulink, etc.) where compilation allows execution without a license, or where licenses minimally restrict the number of running jobs.
```
Comment (djw8605) [inline _about/computation.md:11]: The wording here is awkward.  Sure dynamically linked binaries are fine for more then RHEL 6 or 7, dynamically linked binaries are fine for ubuntu as well, with the right container.  Should you include anything about dynamically linked things at all, if containers solve that problem mostly.

Comment (lmichael107) [reply _about/computation.md:11]: How would you put it?

**Comment by lmichael107** [comment]
see #74

**Pull request closed without merging**